virtcontainers/qemu.go

```diff
@@ -837,6 +852,61 @@ func (q *qemu) hotplugRemoveCPUs(amount uint32) (uint32, error) {
 	return amount, q.sandbox.storage.storeHypervisorState(q.sandbox.id, q.state)
 }
 
+func (q *qemu) hotplugMemory(slot uint32, memMB int, op operation) error {
+	if memMB <= 0 {
+		return errors.New("cannot hotplug zero or negative MB memory")
```

Comment:
Could you make the `memMB` a `uint64` to avoid having to add this check? Same question for the function below.

Reply:
Sure. But I'm keeping the one in `q.hotplugAddMemory` since the govmm requires `int` and we need a sanity check better early than later.